e2e-app/src/app/tools.po.ts

```diff
@@ -129,5 +99,5 @@ export const mouseMove = async(selector: string) => {
   const rect = await getBoundingBox(selector);
```

Comment:
This function is not needed anymore, it's now better to use [ locator.hover](https://playwright.dev/docs/api/class-locator#locator-hover)